Bottom: 90732a346985144bed557ac18ddd1e7d31460241
Top:    71ffaf71df908ab80165b6ee242cbebbc47d220e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-28 21:28:08 +0100

parse_y.y: Don't bother setting LayerFlag[1,2] for elements - it isn't used


---

diff --git a/src/misc.c b/src/misc.c
index e2644f8..0b756bd 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -994,6 +994,8 @@ ParseGroupString (char *s, LayerGroupType *LayerGroup, int LayerN)
     s_set = false;              /* provide a default setting for old formats */
   int groupnum[MAX_LAYER + 2];
 
+  printf ("ParseGroupString (\"%s\", %p, %i)\n", s, LayerGroup, LayerN);
+
   /* clear struct */
   memset (LayerGroup, 0, sizeof (LayerGroupType));
 
@@ -1044,6 +1046,8 @@ ParseGroupString (char *s, LayerGroupType *LayerGroup, int LayerN)
           LayerGroup->Entries[group][member++] = layer;
           while (*++s && isdigit ((int) *s));
 
+          printf ("Group %i contains Layer %i\n", group, layer);
+
           /* ignore white spaces and check for separator */
           while (*s && isspace ((int) *s))
             s++;
diff --git a/src/parse_y.y b/src/parse_y.y
index e6ce092..c6f5246 100644
--- a/src/parse_y.y
+++ b/src/parse_y.y
@@ -226,8 +226,6 @@ parsepcb
 		    PCBType *pcb_save = PCB;
 		    ElementType *e;
 
-		    LayerFlag[0] = true;
-		    LayerFlag[1] = true;
 		    yyData->LayerN = 2;
 
 		    CreateNewPCBPost (yyPCB, 0);
